cfg/dn_rules.sh

```diff
@@ -49,8 +49,6 @@ ebtables -t nat -A POSTROUTING -d Multicast --mark 0x1 -j DROP
 
 #Accept ARP request on eth0 (untagged/tagged interfaces) and don't copy it to NFLOG
 ebtables -A OUTPUT -p ARP -o eth0+ --arp-op Request -j ACCEPT
-#NFLOG to copy all ARP requests to netlink group 100
-ebtables -A OUTPUT -p ARP --arp-op Request --nflog-group 100 -j DROP
```

Comment:
Can u plz add a little detail about the root cause of this issue and how it is affecting the pings.

Reply:
Added the comment on the issue link:  https://github.com/open-switch/opx-nas-l3/issues/24   #24